src/redux/configureStore.jsx

```diff
@@ -1,16 +1,17 @@
-import { createStore, combineReducers, applyMiddleware } from 'redux';
+import {
+  createStore, combineReducers, applyMiddleware, compose,
+} from 'redux';
```

Comment:
As mentioned in the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/5801583dd2ad132a9326d4bd429291d954e4e824/bookstore/connec_to_api.md?plain=1#L33), kindly use the   [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) from the redux-toolkit instead. This will among other things simplify the import.